src/tests/unit/tests/DetailsView/components/__snapshots__/save-assessment-factory.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`SaveAssessmentFactory getSaveButtonForAssessment renders save assessment button 1`] = `
 <SaveAssessmentButton
-  download="filename"
+  download="SavedAssessment_<undefined>_title.a11yextension"
```

Comment:
The `<undefined>` here looks a little off; it suggests that either the test is missing setting something up, or the real code is misusing the file builder.